deepchem/lightning/dc_lightning_module.py

```diff
@@ -0,0 +1,41 @@
+import pytorch_lightning as pl
+import torch
+
+
+class DCLightningModule(pl.LightningModule):
```

Comment:
Please add a docstring detailing the purpose of the class along with a docstring. See `TorchModel`'s docstring for an example. The docstring should be a couple of paragraphs long and detailed.